ai-data/llm-inference/concepts.mdx

```diff
@@ -0,0 +1,70 @@
+---
+meta:
+  title: LLM Endpoints - Concepts
+  description: This page explains all the concepts related to Inference for LLMs
+content:
+  h1: LLM Endpoints - Concepts
+  paragraph: This page explains all the concepts related to Inference for LLMs
+tags: 
+categories:
+  - ia
+---
+## Allowed IPs
```

Comment:
```suggestion
## Access Control List rules
```

Reply:
It's called Allowed IPs in our Console, so I think we should stay consistent with the concept na
<img width="1185" alt="Capture d’écran 2024-03-19 à 15 29 24" src="https://github.com/scaleway/docs-content/assets/4608664/548963f4-026b-4851-8316-f73b8663811c">
ming